core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -1189,8 +1187,8 @@ public RelBuilder filter(Iterable<? extends RexNode> predicates) {
     return filter(ImmutableSet.of(), predicates);
   }
 
-  /** Creates a {@link Filter} of a list of correlation variables
-   * and an array of predicates.
+  /** Creates a {@link Filter} with the specified correlation variables
+   * and predicates.
```

Comment:
Just curious, is `with the specified` better than `of a list of` ?

Reply:
I would say yes for the two reasons below:
1. - `a list of` implies a specific implementation (List) but the method can accept many more since it takes an Iterable.
2. - `specified argument` is used extensively in the doc of JDK. 
Bu still if you believe that it was better before I can rollback the changes, I don't mind.

Reply:
Great, thanks for your explanation.

Reply:
I prefer "list of". It's plainer english, less lawyerly. (If "Iterable" were an english word, I'd say use that. But it isn't.)

Also, I believe that two overloads of the same method should have different descriptions. We should not have two methods both with the description "Creates a {@link Filter} with the specified predicates."

Reply:
Sounds good, I will revert back these and other related doc changes!